api/errors.go

```diff
@@ -93,3 +93,8 @@ type Error string
 
 // Error returns the error message.
 func (e Error) Error() string { return string(e) }
+
+// Webhook errors
+const (
+	ErrWebhookAlreadyExists = Error("A webhooks with the same name already exists")
```

Comment:
The error message should be updated to something else. There is no name associated to a webhook, there is a service ID though.